pydantic/json_schema.py

```diff
@@ -803,7 +803,9 @@ def list_schema(self, schema: core_schema.ListSchema) -> JsonSchemaValue:
         self.update_with_validations(json_schema, schema, self.ValidationsMapping.array)
         return json_schema
 
-    def tuple_positional_schema(self, schema: core_schema.TuplePositionalSchema) -> JsonSchemaValue:
+    @deprecated('tuple_positional_schema is deprecated. Use tuple_schema instead.')
+    @final
+    def tuple_positional_schema(self, schema: core_schema.TupleSchema) -> JsonSchemaValue:
         """Generates a JSON schema that matches a positional tuple schema e.g. `Tuple[int, str, bool]`.
```

Comment:
Do we want to change this docstring at all based on the changes to the logic of the function? Is it necessarily **positional**?

Reply:
Will just axe the docstring for simplicity.